airflow/operators/subdag_operator.py

```diff
@@ -33,6 +37,14 @@
 from airflow.utils.state import State
 
 
+class SkippedStatePropagationOptions(Enum):
+    """
+    Available options for skipped state propagation of subdag's tasks to parent dag tasks.
+    """
+    ALL_LEAVES = 1
```

Comment:
I thought we always start at 0 in the software world LOL

Reply:
oh ya we do (that is embarrassing) 😅 I'll change that.

Reply:
Just joking. no need to change

Reply:
Maybe it is better to have string values.

```
    ALL_LEAVES = 'all_leaves'
    ANY_LEAF = 'any_leaf'
```

WDYT?

Reply:
I think string values would be a better option

Reply:
👍